app/setupPreloadAPIs.js

```diff
@@ -119,6 +119,15 @@ const setupPreloadedSafeAuthAPIs = ( store ) =>
 
             if ( theCall.done && callPromises.resolve )
             {
+                if ( theCall.name === 'login' )
+                {
+                    logger.info('store subscribe calls: ', calls);
+                    logger.info('pendingCalls: ', pendingCalls);
+                    logger.info('call Promises: ', callPromises);
+		    // QUESTION: callPromises.resolve logs `null` \
```

Comment:
Gooood Q. It should be....

Does this happen all the time? With a specific call? If we're not getting into the above condition then something's wrong for sure.

Maybe we should just be checking for `pendingCalls[theCall.id]`... although not sure if that'll make a difference.

Reply:
Yeah, the condition is met every time I login to authenticator.

I just don't understand how `theCall.done && callPromises.resolve` is met when `callPromises.resolve` is always null.